.github/workflows/build.yml

```
@@ -2,8 +2,9 @@ name: build
 
 on:
   push:
-    branches:
-      - "main"
+    branches-ignore:
+      - docs/**
+      - backport/docs/**
```

We don't run builds tests for branches prefixed by E2E, as those changes only impact our nightly E2E suite (and takes 1hr+):

```suggestion
    branches-ignore:
      - docs/**
      - backport/docs/**
      - e2e-*
```